---a/net/minecraft/world/MinecraftException.java
+++b/net/minecraft/world/MinecraftException.java
@@ -1,12 +1,11 @@
 package net.minecraft.world;
 
-public class MinecraftException extends Exception
-{
+public class MinecraftException extends Exception {
+
     private static final String __OBFID = "CL_00000145";
 
-    public MinecraftException(String msg)
-    {
+    public MinecraftException(String msg) {
         super(msg);
     }
-}
+}
